src/libraries/System.Net.Security/tests/StressTests/SslStress/Readme.md

```diff
@@ -20,7 +20,7 @@ $ dotnet run -- -help
 
 Note that the stress suite will test the sdk available in the environment,
 that is to say it will not necessarily test the implementation of the local runtime repo.
-To achieve this, you will need to point your environment to the [`testhost` build](https://github.com/dotnet/runtime/blob/master/docs/coreclr/building/testing-with-corefx.md).
+To achieve this, you will need to point your environment to the [`testhost` build](https://github.com/dotnet/runtime/blob/master/docs/workflow/building/coreclr/testing-with-corefx.md).
```

Comment:
This doc does not exist

Reply:
Good catch; thanks. I’ll find the updated page and push another commit.

Reply:
I’m actually unsure what this doc should say now. @eiriktsarpalis - can you suggest an update to this section?

Reply:
```suggestion
To achieve this, we will first need to build a new sdk from source. This can be done [using docker](https://github.com/dotnet/runtime/blob/master/eng/docker/Readme.md).
```

Reply:
I haven't been able to find a good substitute, so I've just replaced it with the specific docker build approach that the pipelines are using.